packages/public/common/src/hooks/useObjectTranslation.ts

```diff
@@ -0,0 +1,26 @@
+import { MonkLanguage, TranslationObject } from '@monkvision/types';
+import { useTranslation } from 'react-i18next';
+
+/**
+ * The result of the useObjectTranslation. It contains a function which takes a LabelTranslation object and return the
+ * translated label sync with the actual selected language.
+ */
+export interface UseObjectTranslationResult {
+  /**
+   * Function translating a LabelTranslation object into a translated label sync with the actual selected language.
+   * @param obj
+   */
+  tObj: (obj: TranslationObject) => string;
+}
+
+/**
+ * Custom hook used to get the label with the actual selected language.
```

Comment:
Change doc to : `Custom hook used to get a translate function (\`tObj\`) that translates TranslationObjects.`